index.d.ts

```diff
@@ -133,6 +133,38 @@ type ReaderRevenuePosition =
     | 'ampHeader'
     | 'ampFooter';
 
+interface CampaignsType {
+    id: string;
+    name: string;
+    rules: [];
+    priority: number;
+    activeFrom: number;
+    displayOnSensitive: boolean;
+    fields: {
+        formId: number;
+        callout: string;
+        _type: string;
+        description: string;
+        tagName: string;
+        formFields: CampaignsFeildType[];
+    };
+}
+
+interface CampaignsFeildType {
+    text_size?: number | string; // TODO: need to fix at source
+    name?: string;
+    description?: string;
+    hide_label?: string;
+    label?: string;
+    id?: string;
+    type: string;
+    required: string;
+    options?: {
+        label: string;
+        value: string;
+    }[];
```

Comment:
It might be clearer if this were defined separately as `CampaignOptionType`.

That said, the name `options` makes no sense in this context. They _look_ more like tags of some sort. Perhaps a comment here describing what this is would help?